lib/error.js

```diff
@@ -212,4 +212,15 @@ AppError.invalidContentType = function invalidContentType() {
   });
 };
 
+AppError.invalidScopes = function invalidScopes(scopes) {
+  return new AppError({
+    code: 400,
+    error: 'Invalid scopes',
+    errno: 114,
+    message: 'Invalid scopes'
+  }, {
+    invalidScopes: scopes
+  });
+};
+
```

Comment:
Please add this new error code to ./docs/API.md (or file a follow-up bug to do it once the pace slows down a little)